site/content/docs/reference/cli.md

```diff
@@ -10,75 +10,141 @@ menu:
     parent: reference
 ---
 
-## `pinniped version`
+## pinniped get kubeconfig
```

Comment:
Note! This doc format is different from what we had previously. My hope is that it is still a sufficient way to document our CLI. There are a lot of benefits on the implementation side - we reuse `"github.com/spf13/cobra/doc.GenMarkdownCustom"` to generate our docs for us, and this is automatically checked in CI.

Reply:
Here is what the docs look like now: 
![image](https://user-images.githubusercontent.com/3081590/112011841-092a6900-8aff-11eb-8021-3a2a89ffd692.png)

Reply:
Here is what the docs look like based on this PR:
![image](https://user-images.githubusercontent.com/3081590/112011942-1fd0c000-8aff-11eb-9b07-284ca791c11c.png)